doc/sphinx/using/libraries/funind.rst

```diff
@@ -412,3 +412,19 @@ Generation of induction principles with ``Functional`` ``Scheme``
          Generate graph for @qualid
 
    Internal debugging commands.
+
+Flags
+-----
+
+.. flag:: Functional Induction Rewrite Dependent
+
+   Makes funind use dependent :tacn:`subst`  instead of :tacn:`simple subst`.
```

Comment:
For consistency with the casing used throughout this chapter:
```suggestion
   Makes FunInd use dependent :tacn:`subst`  instead of :tacn:`simple subst`.
```